common/i18n/resources/en.js

```diff
@@ -245,4 +245,26 @@ module.exports = {
     TaxonomiesValidationJob: 'Taxonomies Validation',
     TaxonomyImportJob: 'Taxonomy Import',
   },
+
+  formErrors: {
+    duplicateName: 'duplicate scientific name {{scientificName}}; row: {{row}} duplicate row: {{duplicateRow}}',
+    duplicateCode: 'duplicate code {{code}}; row: {{row}} duplicate row: {{duplicateRow}}',
```

Comment:
can we use a nested key in this case? something like $t(duplicateRow)...

Reply:
ok done